db/upgrade.php

```diff
@@ -28,6 +29,17 @@ function xmldb_tool_cloudmetrics_upgrade($oldversion) {
 
     $dbman = $DB->get_manager();
 
+    // Upgrade script turn all metrics on.
```

Comment:
general best practice all upgrade chunks should be in order so they execute in version order, so move this down about 15 lines please